seeds.sql

```diff
@@ -1,11 +1,2532 @@
+(7, 'IDP Provider 7', 'idp7.example.com', 'admin7@example.com', '2024-12-22 12:41:45.680', '2024-10-30 12:41:45.680', NULL, 'noreply@idp7.example.com', 'NONE_OR_EMAIL', NULL, NULL, NULL, 'Qualifier-7', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'https://play.google.com/store/apps/details?id=idp7.app', 'https://itunes.apple.com/us/app/idp7', 0, NULL, NULL, NULL, 28, 0, NULL, NULL, 21, 8, NULL, 'fr', '2018-03-22 12:41:45.680', NULL, 5, '#FFFFFF', '#000000', 'https://s3.amazonaws.com/idp7/logo.png', 1),
+(8, 'IDP Provider 8', 'idp8.example.com', 'admin8@example.com', '2024-04-21 12:41:45.680', '2024-09-30 12:41:45.680', NULL, 'noreply@idp8.example.com', 'SHIBBOLETH', NULL, NULL, NULL, 'Qualifier-8', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'https://play.google.com/store/apps/details?id=idp8.app', 'https://itunes.apple.com/us/app/idp8', 0, NULL, NULL, NULL, 60, 0, NULL, NULL, 21, 25, NULL, 'fr', '2023-01-05 12:41:45.680', '2025-03-21 12:41:45.680', NULL, NULL, '#F5F5F5', 'https://s3.amazonaws.com/idp8/logo.png', 1),
+(9, 'IDP Provider 9', 'idp9.example.com', 'admin9@example.com', '2024-12-08 12:41:45.680', '2024-10-03 12:41:45.680', 'OLD', 'noreply@idp9.example.com', 'SESSION_SHARING', NULL, NULL, NULL, 'Qualifier-9', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'https://play.google.com/store/apps/details?id=idp9.app', 'https://itunes.apple.com/us/app/idp9', 0, NULL, NULL, NULL, 97, 1, NULL, NULL, 25, 25, NULL, 'es', '2021-02-01 12:41:45.680', '2025-12-10 12:41:45.680', NULL, NULL, NULL, 'https://s3.amazonaws.com/idp9/logo.png', 0),
+(10, 'IDP Provider 10', 'idp10.example.com', 'admin10@example.com', '2024-05-22 12:41:45.680', NULL, 'ORIG-ORCA', 'noreply@idp10.example.com', 'EMAIL', NULL, NULL, NULL, 'Qualifier-10', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'https://play.google.com/store/apps/details?id=idp10.app', 'https://itunes.apple.com/us/app/idp10', 0, NULL, NULL, NULL, 99, 0, NULL, NULL, 16, 14, NULL, 'fr', '2023-04-02 12:41:45.680', '2025-09-20 12:41:45.680', 7, '#F5F5F5', '#FFFFFF', 'https://s3.amazonaws.com/idp10/logo.png', 1);
 
-INSERT INTO `user` (`id`, `user_id_from_idp`, `idp_id`, `email`, `fullname`, `adminLevel`, `created_at`, `last_login_at`, `last_login_platform`)
-VALUES
-	(3, 'dev@toadreader.com', 21, 'dev@toadreader.com', 'Mr. Dev', 'ADMIN', '2020-10-14 16:47:50.859', '2024-02-29 21:32:48.023', 'Chrome 122.0.0 / Mac OS X 10.15.7');
+INSERT INTO `user` (`id`, `user_id_from_idp`, `idp_id`, `email`, `fullname`, `adminLevel`, `created_at`, `last_login_at`, `last_login_platform`) VALUES
```

Comment:
As we are using Shibboleth, I would expect that the user table can be blank and be populated when a user logs in. If you want to prepoulate it, please ensure that it will work with [this test data](https://github.com/biblemesh/ereader-callback/blob/d07fe9bbb8b4695261eeb308b48935cd498322c2/src/database/user.ts#L7).